docs/containerizedcomputation.md

```diff
@@ -1,18 +1,17 @@
 ### Start the service:
-- Run `docker compose up funnel`
+- Run `docker compose --project-directory config/gdi-starter-kit up funnel -d`
 - To try it with the rest of the stack, also run:
-`docker compose up htsgetDecrypt`
+`docker compose --project-directory config/gdi-starter-kit up htsgetDecrypt -d`
 
 
 ### Access the service:
-- Go into the directory: `cd containerized-computation`
 - Get a token:
     `token=$(curl -s -k https://localhost:8080/tokens | jq -r '.[0]')`
 - Put it in your task file:
-      `sed -i -e "s/bearer:.*@/bearer:$token@/g" task1.json`
+      `sed -i -e "s/bearer:.*@/bearer:$token@/g" config/gdi-starter-kit/containerized-computation/task1.json`
```

Comment:
This still edits the file in place which may not be what we want from a temple. I would suggest dropping th `-i` flag and piping to a new file in the working folder.

Reply:
in c42edaf